eng/CredScanSuppression.json

```diff
@@ -16,7 +16,9 @@
                 "dummyPassword",
                 "P@ssw0rd",
                 "12NewPA$$w0rd!",
-                "Samp1eP@ssw0rd"
+                "Samp1eP@ssw0rd",
+                "facebookapplicationsecret",
+                "updatedfacebooksecret"
```

Comment:
I see "StrongPass!123" used in [resourcemanager](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-authorization/src/test/java/com/azure/resourcemanager/authorization/ServicePrincipalsTests.java#L165). Is credscan not complaining about this?

Reply:
Appears it doesn't, or it is part of the baseline suppression

Reply:
Any reason we need to add additional placeholders as opposed to using an existing one?

Reply:
> Any reason we need to add additional placeholders as opposed to using an existing one?

These were in code samples in a library I don't have any oversite on, so I thought it would be better to add new suppression placeholders than update samples I don't have context into.

Reply:
In general we are trying to reduce and limit the number of place holders so as we tweak things we should try to do that over adding more.